### PR TITLE
add chainweb events

### PR DESCRIPTION
**Pull request by ggobugi27**
- Add event query functions: 
`Pact.event.range`, `Pact.event.recent`, `Pact.event.stream`, `Pact.event.height`, `Pact.event.blockhash`
- Update readme
- Add tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
     "blakejs": "^1.0.1",
     "blue-tape": "^1.0.0",
     "browserify": "^16.5.1",
-    "node-fetch": "^2.6.0",
+    "eventsource": "^1.1.0",
+    "jest": "^27.4.3",
```
Comment (larskuhtz) [inline package.json]: I think, npm allows to distinguish between prod and test dependencies. Though not sure.

File: pact-lang-api.js [hunk 2]
```diff
@@ -823,6 +823,689 @@ const sendSigned = async function (signedCmd, apiHost) {
   return tx;
 }
 
+
+/**
+ * Functions for querying the Kadena Chainweb blockchain API
+ *
+ * @author Lars Kuhtz
+ * @module chainweb
```
Comment (larskuhtz) [inline pact-lang-api.js]: I think, those are module level comment, but here the code got copied into a different module

Comment (larskuhtz) [reply pact-lang-api.js]: Would it make sense for better maintainability to add it as a separate module to the project? And maybe re-export in pact-lang-api.js

**Review by larskuhtz — COMMENTED** [review]
I think, for code maintainability, it would be better to copy the complete module from chainweb.js as a module and import and re-export the events API endpoints in pact-lang-api.js.

File: package.json (hunk 1 above)
Comment (LindaOrtega) [reply package.json]: I moved test dependencies into a "devDependencies" section. This stackoverflow answer goes into the difference sections: https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

File: pact-lang-api.js (hunk 2 above)
Comment (LindaOrtega) [reply pact-lang-api.js]: I imported `chainweb.js` as a dependency into `pact-lang-api.js` and then re-exported the chainweb.js endpoint functions we care about. So there's no need to copy over code from chainweb.js into this repo for now.

**Review by larskuhtz — APPROVED** [review]

**Review by buckie — APPROVED** [review]

**Pull request merged**